spec/models/miq_provision_virt_workflow_spec.rb

```diff
@@ -253,4 +253,24 @@
       expect(workflow.ws_find_template_or_vm("", "VMWARE", "asdf-adsf", "asdfadfasdf")).to be_a(MiqHashStruct)
     end
   end
+
+  context '#update_request' do
+    let(:template) do
+      FactoryGirl.create(
+        :template_vmware,
+        :ext_management_system => FactoryGirl.create(:ems_vmware_with_authentication)
+      )
+    end
+    let(:values)  { {:src_vm_id => [template.id, template.name]} }
```

Comment:
Is there any reason why we can't just use static values here for id and name? I'm just thinking it's probably not necessary to create a whole template and ext management system when all we really care about is the id and name.